Source/CMakeLists.txt

```diff
@@ -862,7 +872,7 @@ endif()
 add_devilutionx_object_library(libdevilutionx ${libdevilutionx_SRCS})
 target_include_directories(libdevilutionx PUBLIC ${CMAKE_CURRENT_BINARY_DIR})
 target_link_dependencies(libdevilutionx PUBLIC
-  Threads::Threads
+  #Threads::Threads

```

Comment:
Should be DOS-only

Reply:
There are still a few bad hacks in here, but I do appreciate suggestions for how to deal with some of them.

Reply:
How do I actually make this one conditional?

Reply:
This line can be removed and the following added instead:

```cmake
if(NOT TARGET_PLATFORM STREQUAL "dos")
  target_link_dependencies(libdevilutionx PUBLIC Threads::Threads)
endif()
```